org/ystia/experimental/consul/linux/ansible/playbooks/templates/3_secure_server_conf.json.j2

```diff
@@ -0,0 +1,25 @@
+{#
+Copyright 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#}
+{
+  "key_file": "{{ CONFIG_DIR }}/consul.key",
+  "cert_file": "{{ CONFIG_DIR }}/consul.pem",
+  "ca_file": "{{ CONFIG_DIR }}/ca.pem",
+  "verify_incoming": true,
+  "verify_outgoing": true,
+  "ports": {
+      "https": 8543
```

Comment:
The port should not be hardocded as it is an endpoint capability property that the user can set in his topology template